YAXLib/Customization/IMemberContext.cs

```diff
@@ -11,19 +9,9 @@ namespace YAXLib.Customization;
 public interface IMemberContext
```

Comment:
`IMemberContext` should be kept compatible with v4.1.0 so there's no need for a new major version

Reply:
Done. Please take a look.